README.md

```diff
@@ -1,63 +1,64 @@
 - All these features give the developers an environment to experiment and make errors without affecting the final working code.
 - All these updates and experiments can easily be reviewed when using a version control tool.
 
-A version control tool provides many more advantages and powerful features to the developer, but one of the biggest advantages of using this tool is that multiple team members can collaborate on a single project and utilize all these features simultaneously.
+A version control tool provides many more advantages and powerful features to the developer, but one of the biggest advantages of using this tool is that multiple team members can collaborate on a single project and utilize all these features simultaneously ([ref?]()).
 
 Now that we've talked about what a version control system is and its advantages, let's look at how to use Git for version control activities.
-- Step 1 - Create a repository.
-You can create a repository for the code on the GitHub platform by following the simple prompts for creating the repository.
+1. Create a repository.
+You can create a repository for the code on the [GitHub](https://www.github.com) platform by following the simple prompts for creating the repository.
 - Step 2 - Clone the repository.
 Once a repository is created, you need to replicate it to your local machine. You can create files and folders in the repository and start writing your code.
 - Step 3 - Save and commit.
 As you write your code, save your files with every small code block change and commit those changes. This is an important step, as each commit becomes a new state of the code that can later be revisited. It's important to create commits with short but descriptive commit titles and descriptions.
-- Step 4 - View commit history.
-As you develop your project, you can keep track of the history of your commits to ensure the changes you've made to the code.
+- ~~Step 4 - View commit history.~~~
+~~As you develop your project, you can keep track of the history of your commits to ensure the changes you've made to the code.~~
 - Step 5 - Make any required changes.
 If you identify that your code requires changes, revisit those parts, make the changes, and recommit your code.
 - Step 6 - Push changes.
 Finally, all these commits need to be pushed to the remote repository. Once in the cloud, you can access your project from any other device.
 
-(Source:  [Open Source](https://opensource.com/article/18/1/step-step-guide-git)).
+(Source:  [Open Source](https://opensource.com/article/18/1/step-step-guide-git)). _\[I question the accuracy of this reference\]_
 
 ## Code Example for incremental commits
 
 Here's a simple example of what it means to save your code with small blocks of code. For instance, let's say you're writing a simple Java program for a calculator that performs tasks such as addition, subtraction, multiplication, and division.
 
+_\[Why are these separate commits?\]_
```

Comment:
I think you've missed a *big* opportunity here. Many people have successfully learned how to commit and push code, but only a small number of those understand that value of _good_ commits and commit messages from the early stages of their careers. I appreciate that you've built a structure in your code examples to help communicate how the code changes - _but none of these commits are really meaningful on their own_ - and you've not demonstrated good commit messages.

You have a whole section below that talks about good practices in commits, but I think these examples don't quite meet those expectations. If you want to really demonstrate a good example of proper commits, read that section below again carefully and consider how you might use those ideas to better demonstrate _small_ but _complete_ changes that make _relative_ sense.